tests/tacacs_server.yaml.example

```diff
@@ -0,0 +1,6 @@
+# Tacacs server configuration for test_aaa_authorization_service.rb
+# Tacacs server must be set up with external access for test to run
+host: '10.122.197.197'
+key:  'testing123'
+vrf:  'management'
+intf: 'mgmt0'
```

Comment:
If I made it the actual `tacacs_server.yaml` file, git would want to track the file any time we updated the server info.  It would make it easy to accidentally overwrite the file with a commit of someone's server info.